book/website/welcome.md

```diff
@@ -1,22 +1,28 @@
+(welcome)=
 # Welcome
 
-_The Turing Way_ is an open source community-driven guide to reproducible, ethical, inclusive and collaborative data science.
+_The Turing Way_ is an open-source community-driven guide to reproducible, ethical, inclusive and collaborative data science.
```

Comment:
```suggestion
_The Turing Way_ is an open source community-driven guide to reproducible, ethical, inclusive and collaborative data science.
```

Reply:
We use open source without hyphenation, though I know Grammarly doesn't understand! :D I'd suggest ignoring Grammarly suggestion on this one.

Reply:
Thanks! I will keep this in mind!